src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -36,12 +40,24 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Cyotek.Drawing.BitmapFont, Version=1.0.2.0, Culture=neutral, PublicKeyToken=58daa28b0b2de221, processorArchitecture=MSIL">
```

Comment:
Thanks for catching this one - please add them to the 3rd party tracking sheet. See me for the link.